### ISSUE TITLE
detection is functioning but my aim did not lock on the enemy

### ISSUE THREAD

**Issue by tytdoge** [NTUYWANG103/APEX_AIMBOT#60]
old version Ghub was installed (2021.3) but the aimbot is still not working, any additional configuration required?
![image](https://github.com/NTUYWANG103/APEX_AIMBOT/assets/111518624/e0f7afdb-4309-4010-a968-7d55bbfd48c2)

**Comment by tytdoge** [comment]
![image](https://github.com/NTUYWANG103/APEX_AIMBOT/assets/111518624/9729aebe-f83c-446e-a15a-58e04950614a)

**Comment by Thiece** [comment]
After installing Ghub, please restart your computer after launching Ghub once

**Comment by Gaius-Julius-Caeasar** [comment]
Must it be a Logitech mouse？

**Comment by Thiece** [comment]
> Must it be a Logitech mouse？

No, what's needed are the drivers brought by Ghub, it has nothing to do with whether you are using a Logitech mouse or not.
Please note the mentioned range of Ghub versions, after installation you need to launch Ghub once and restart the computer once.
